### DISCUSSION TITLE
Can it be wired to the Sprite Extruder's board, using Klipper?

### DISCUSSION THREAD

**Discussion by drewgcolemancwg** [kevinakasam/KlackEnder-Probe#37] [General]
Is it possible to wire this to the breakout board on the Ender 3 S1 toolhead? I rather don't like the CR Touch at this point (just have had some inconsistent readings for various reasons), plus I love to tinker. Trying to figure out if I can just wire it to the existing connection, and then configure Klipper.

Looking at the wiring diagram here:

https://support.dremc.com.au/support/solutions/articles/51000298817-ender-3-s1-s1-pro-breakout-pin-out

I think I can just crimp in a connector for pins 13 and 15 (Z_Probe and Ground). Does that sound right?

**Comment by Laph9979** [comment]
Any follow up on this? Looking to do the same.
